client/flags/flags.go

```diff
@@ -35,6 +35,10 @@ const (
 	SignModeLegacyAminoJSON = "amino-json"
 	// SignModeDirectAux is the value of the --sign-mode flag for SIGN_MODE_DIRECT_AUX
 	SignModeDirectAux = "direct-aux"
+	// SignModeTextual is the value of the --sign-mode flag for SIGN_MODE_TEXTUAL.
+	// Choosing this flag will result in an error for now, as Textual should be
+	// used only for TESTING purposes for now.
+	SignModeTextual = "textual"
```

Comment:
Does it make sense to add this flag now, which shows an error upon usage? We could also just add CLI flags later on